samples/bluetooth/peripheral_bms/README.rst

```diff
@@ -0,0 +1,102 @@
+.. _peripheral_bms:
+
+Bluetooth: Peripheral Bond Management Service (BMS)
+###################################################
+
+The peripheral BMS sample demonstrates how to use the :ref:`bms_readme`.
+
+Overview
+********
+
+When connected, the sample waits for Client requests to perform any bond deleting operation.
+
+It supports up to two simultaneous Client connections.
+
+Requirements
+************
+
+* One of the Nordic development boards:
+
+  * |nRF5340DK|,
+  * |nRF52840DK|,
+  * |nRF52DK|.
+
+* A Bluetooth Low Energy dongle or development kit.
+
+* nRF Connect for Desktop.
+
+User interface
+**************
+
+LED 1:
+   * It blinks with a period of 2 seconds with the duty cycle set to 50% when the main loop is running, and the device is advertising.
+
+LED 2:
+   * It is on when connected.
+
+Building and Running
+********************
+.. |sample path| replace:: :file:`samples/bluetooth/peripheral_bms`
+
+.. include:: /includes/build_and_run.txt
+
+.. _peripheral_bms_testing:
+
+Testing
+=======
+
+After programming the sample to your development board, test it by performing the following steps:
+
+1. |connect_terminal_specific|
+#. Reset the board.
+#. Start `nRF Connect for Desktop`_ and select the connected device that is used for communication.
+#. Connect to the device from nRF Connect.
+   The device is advertising as "Nordic_BMS".
+#. Bond to the device.
+   To bond:
+   
+   a. click the settings button for the device in nRF Connect,
```

Comment:
Better make these points into standalone sentences with capital letters and full stops.
This structure with commas and fulls stop after only the last point is good when you want to enumerate some items, or for example to present different options. But when it comes to actual steps or tasks in a procedure, it should definitely be standalone sentences.

Reply:
ok, will keep that in mind for the next PRs. Fixed